tests/integration/test_remoted/test_agent_communication/test_invalid_protocol_communication.py

```diff
@@ -1,3 +1,56 @@
+'''
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'wazuh-remoted' program is the server side daemon that communicates with the agents.
+       Specifically, these tests will check if the agent status appears as 'disconnected' after
+       just sending the 'start-up' event, sent by several agents using different protocols.
+       The 'disconnected' status is when the manager considers that the agent is disconnected
+       if it does not receive any keep alive messages.
```

Comment:
This is the brief of `tests/integration/test_remoted/test_agent_communication/test_agent_version_shared_configuration_startup_message.py` module

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/2426/commits/d93265687f322da854aca0df2364ee986a1c8be3)